### DISCUSSION TITLE
Example-like resizable panel

### DISCUSSION THREAD

**Discussion by sammtan** [shadcn-ui/ui#3781] [General]
Does anybody know how to create the same panel and its behavior like the one example in official website?
![Example](https://github.com/shadcn-ui/ui/assets/127273133/d819007c-f69b-488f-a3ea-343a74375c17)
It doesn't have to be exactly same; I just want to have the panel, resize ability and its handle, and the responsive content inside. I did some research, but I got no related solution.

**Comment by sammtan** [comment]
nvm, I found the solution. thanks!

**Reply by 0xtz** [reply]
I was about to post the code ... bc I created one bc no resources and I saw your comment :joy:
